app/Http/Livewire/Comment.php

```diff
@@ -142,14 +150,46 @@ final public function postReply(): void
         }
 
         // New Comment Notification
-        if ($this->user->id !== $this->comment->user_id) {
+        if ($this->user->id !== $this->comment->user_id && strtolower(class_basename($this->comment->commentable_type)) !== 'collection') {
             User::find($this->comment->user_id)->notify(new NewComment(strtolower(class_basename($this->comment->commentable_type)), $reply));
         }
 
         // User Tagged Notification
-        if ($this->user->id !== $this->comment->user_id) {
-            $users = User::whereIn('username', $this->taggedUsers())->get();
-            Notification::sendNow($users, new NewCommentTag(strtolower(class_basename($this->comment->commentable_type)), $reply));
+        $users = User::whereIn('username', $this->taggedUsers())->get();
+        Notification::sendNow($users, new NewCommentTag(strtolower(class_basename($this->comment->commentable_type)), $reply));
+
+        // Auto Shout
+        $profileUrl = href_profile($this->user);
+
+        $modelUrl = match (strtolower(class_basename($this->comment->commentable_type))) {
+            'article'    => href_article($this->comment->commentable_type),
+            'collection' => href_collection($this->comment->commentable_type),
+            'playlist'   => href_playlist($this->comment->commentable_type),
+            'request'    => href_request($this->comment->commentable_type),
+            'torrent'    => href_torrent($this->comment->commentable_type),
+            default      => "#"
+        };
+
+        if (strtolower(class_basename($this->comment->commentable_type)) !== 'ticket') {
```

Comment:
Probably should add code so that if a user replies to their own parent comment in a ticket, the staff member assigned the ticket still gets notified.